engine/link/link.asm

```diff
@@ -217,17 +217,17 @@ Gen2ToGen2LinkComms:
 	call DelayFrames
 	xor a
 	ldh [hSerialSend], a
-	ld a, (0 << rSC_ON) | (1 << rSC_CLOCK)
+	ld a, 0 | SCF_SOURCE
 	ldh [rSC], a
-	ld a, (1 << rSC_ON) | (1 << rSC_CLOCK)
+	ld a, SCF_START | SCF_SOURCE
 	ldh [rSC], a
 
 	call DelayFrame
 	xor a
 	ldh [hSerialSend], a
-	ld a, (0 << rSC_ON) | (1 << rSC_CLOCK)
+	ld a, 0 | SCF_SOURCE
```

Comment:
```suggestion
	ld a, 0 | SCF_SOURCE ; Stop any transfer in progress
```

Reply:
Is the '0 |` part intentional?

Reply:
Nope, I had to slip up once out of thirty-ish times :P